tools/onnx/gen_diagnostics.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cd ../../
```

Comment:
Probably make this more robust?

Reply:
done, added a few lines of comments too.